charts/grafana/templates/_helpers.tpl

```diff
@@ -141,6 +141,17 @@ Return the appropriate apiVersion for ingress.
   {{- end -}}
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for podsecuritypolicy and PodDisruptionBudget.
+*/}}
+{{- define "grafana.policy.apiVersion" -}}
+  {{- if and (.Capabilities.APIVersions.Has "policy/v1") (semverCompare ">= 1.21-1" .Capabilities.KubeVersion.Version) -}}
+    {{- print "policy/v1" -}}
+  {{- else -}}
+    {{- print "policy/v1beta1 -}}
```

Comment:
Missing close double quote here. Should be:
```
{{- print "policy/v1beta1" -}}
```

Reply:
@zanhsieh Fixed! Thank you :)